### DISCUSSION TITLE
[Select] I can use textValue in Item

### DISCUSSION THREAD

**Discussion by mnich** [radix-ui/primitives#2081] [Help] [answered]
Hello everyone,

I would like to display a group and label on a selected item. It always returns me the contents of value

```
 <Styled.SelectItemList>
    {group.items && group.items.map((el, i) => (
        <Styled.SelectItem key={i} value={el.value} textValue={`${group.label} ${el.label}`}>
            <Styled.SelectItemText>
                {el.label}
            </Styled.SelectItemText>
        </Styled.SelectItem>
    ))}
</Styled.SelectItemList>
```

Maybe someone already used textValue and encountered this problem

**Comment by joaom00** [comment, marked as answer]
If you want to show the group and label in the trigger you should render the group with the label in `Styled.SelectItemText`.

https://www.radix-ui.com/docs/primitives/components/select#itemtext

**Reply by mnich** [reply]
Its works, thx

**Comment by benoitgrelard** [comment]
`textValue` is used only used as an override to the textual content present in `Select.ItemText`, this is used for the typeahead logic.

Where do you want to show `${group.label} ${el.label}`?
- In the items (when the menu is open)?
- In the trigger?
- in both?

They are all possible but the answer will vary based on what you are after.

**Comment by mnich** [comment]
In the trigger

**Reply by benoitgrelard** [reply]
Then have a look at this section in the docs: https://www.radix-ui.com/docs/primitives/components/select#controlling-the-value-displayed-in-the-trigger
